tests/torch/test_hybrid_converter.py

```diff
@@ -53,12 +53,13 @@ def run_hybrid_model_test(
 
     # Compute average of these counts (the accuracy)
     accuracy = is_in.float().mean()
+    del accuracy
```

Comment:
why this del ? 😅

Reply:
Just to avoid my linter from saying that the variable was unused.

Reply:
ah yes ok got it

Reply:
actually not enough to have pcc happy